exercises/beer-song/lib/example.dart

```diff
@@ -0,0 +1,29 @@
+class BeerSong {
+  List<String> recite(int start, int numVerses) {
+    List<String> verses = new List();
```

Comment:
For Dart 2, you can set verses as `final`, instead of `List<String>`.

And instead of `new List()`, you can use `<String>[]`.